docs/ensnode.io/src/styles/starlight.css

```
@@ -452,15 +452,46 @@ ul.top-level details details > summary .group-label span {
-/* Align "tablist" component to the left
- * and remove the default full-width underline
- */
+/* Unify the styling of Starlight tabs with our Omnigraph Static example tabs */
 starlight-tabs ul[role="tablist"] {
-  padding-left: 0;
   border-bottom: none;
+  margin-top: 0;
+  padding-left: 20px;
+  padding-top: 12px;
+
+  display: flex;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)